clients/fides-js/src/lib/initialize.ts

```diff
@@ -144,7 +139,7 @@ export const getInitialCookie = ({ consent, options }: FidesConfig) => {
 /**
  * If saved preferences are detected, immediately initialize from local cache
  */
-export const getInitialFides = ({
+export const getInitialFidesFromConsent = ({
```

Comment:
nit: FWIW, the name `getInitialFidesFromConsentCookie` might be more accurate, since this function now mostly just unpacks the `cookie` prop.